src/nodes.coffee

```diff
@@ -1377,6 +1368,34 @@ exports.Value = class Value extends Base
       jisonLocationDataToAstLocationData(@properties[@properties.length - 1].locationData)
     )
 
+exports.MetaProperty = class MetaProperty extends Base
+  constructor: (@meta, @property) ->
+    super()
+
+  children: ['meta', 'property']
+
+  checkValid: (o) ->
+    if @meta.value is 'new'
```

Comment:
When reading about `MetaProperty`, I came across [`import.meta`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta). So makes sense to make `MetaProperty` a little bit "generic" so that we can plan on adding other meta-properties